packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -134,6 +135,12 @@ abstract class CreateBase extends FlutterCommand {
           'This is only intended to enable testing of the tool itself.',
       hide: !verboseHelp,
     );
+    argParser.addOption(
```

Comment:
This is used in the other half of the feature where create is called programmatically to create an app to diff against. We want to specify this so that the metadata around the revisions is using the app's metadata, not creating brand new metadata as a regular create run would do.

Reply:
If this only supposed to be called re-entrantly, let's make it `hide: true` and specify in the help text this is not supposed to be called by users.

Reply:
`hide: !verboseHelp` seems correct.
https://github.com/flutter/flutter/blob/3e43c3e11072eea99fa357776299c20a6b2212c6/packages/flutter_tools/test/general.shard/args_test.dart#L67-L69